devel/cluster/export-logs.sh

```diff
@@ -22,9 +22,13 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
 export REPO_ROOT="${SCRIPT_ROOT}/.."
 source "${SCRIPT_ROOT}/../lib/lib.sh"
 
+IS_OPENSHIFT=${IS_OPENSHIFT:-"false"}
+
 # Require helm available on PATH
 check_tool kind
 
 LOGS_DIR="${ARTIFACTS:-$REPO_ROOT/_artifacts}/cert-manager-e2e-logs"
 rm -rf $LOGS_DIR && mkdir -p $LOGS_DIR
-kind export logs $LOGS_DIR --name "${KIND_CLUSTER_NAME}"
+if [[ "$IS_OPENSHIFT" == "false" ]] ; then
+  kind export logs $LOGS_DIR --name "${KIND_CLUSTER_NAME}"
+fi
```

Comment:
Can be a follow up but it'd be good to find some equivalent we can use for OpenShift (even if it is manual 😓)

Reply:
Added a very ugly hack